src/utils/helpers.js

```diff
@@ -20,3 +20,28 @@ export default function manipultateLinks(tableData) {
   });
   return tableData;
 }
+
+export function dateTimeStamp() {
+  const date = new Date();
+  const yyyy = date.getFullYear();
+  let dd = date.getDate();
+  let mm = (date.getMonth() + 1);
+
+  if (dd < 10) { dd = `0${dd}`; }
+
+  if (mm < 10) { mm = `0${mm}`; }
+
+  const todaysDate = `${yyyy}-${mm}-${dd}`;
+
+  let hours = date.getHours();
+  let minutes = date.getMinutes();
+  let seconds = date.getSeconds();
+
+  if (hours < 10) { hours = `0${hours}`; }
+
+  if (minutes < 10) { minutes = `0${minutes}`; }
+
+  if (seconds < 10) { seconds = `0${seconds}`; }
+
+  return `_${todaysDate}_${hours}-${minutes}-${seconds}${'.csv'}`;
```

Comment:
Can you append .csv in the respective files, so this will be a generic function to get the datetimestamp

Reply:
Good Point. Updated accordingly